SpotifySearch/SpotifySearch/AlbumTableViewController.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  AlbumTableViewController.swift
+//  SpotifySearch
+//
+//  Created by Edward Anchundia on 10/28/16.
+//  Copyright © 2016 Edward Anchundia. All rights reserved.
+//
+
+import UIKit
+
+class AlbumTableViewController: UITableViewController {
+
+    internal let albumTableViewCellIdentifier: String = "AlbumCellIdentifier"
+    internal var albums: [Album] = []
+    internal let albumEndPoint: String = "https://api.spotify.com/v1/search?q=blue&type=album&limit=50"
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.getAlbums(apiEndPoint: albumEndPoint) { (albums: [Album]?) in
+            if albums != nil {
+                    
+                DispatchQueue.main.async {
+                    self.albums = albums!
```

Comment:
You can set `self.albums = albums` above this and outside of the `DispatchQueue.main.async{}` closure. Reserve that closure for UI updates.